Jenkinsfile_CNP

```diff
@@ -0,0 +1,27 @@
+#!groovy
+
+@Library("Infrastructure")
+import uk.gov.hmcts.contino.GradleBuilder
+
+def type = "java"
+def product = "reform-scan"
+def component = "blob-router"
+
+def channel = '#bsp-build-notices'
+GradleBuilder builder = new GradleBuilder(this, product)
+
+withPipeline(type, product, component) {
+  after('test') {
+    builder.gradle('integration')
+  }
+
+  after('smoketest:aat') {
+    sh "rm -rf infrastructure/"
+  }
+
+  enableDockerBuild()
+  enableAksStagingDeployment()
+  installCharts()
+  enableSlackNotifications(channel)
+  disableLegacyDeployment()
```

Comment:
do we need this ? It was never deployed on ASE ? but never know for DNS we may need it